save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/DemoRunComponent.kt

```diff
@@ -42,6 +43,8 @@ private val defaultCode = """
     |
     |fun main() {
     |    val SAVEOUR = "tool"
+    |    
```

Comment:
do we need it? Or is it to have a formatting issue in stub example?

Reply:
Yep, it is to make `ktlint` find some formatting issues